lib/Widgets/MessageDialog.vala

```diff
@@ -333,6 +335,6 @@ public class Granite.MessageDialog : Granite.Dialog {
             }
         }
 
-        details_view.label = error_message;
+        details_view.append_text (error_message);
```

Comment:
This should still be a set/replaces. Can't change API out from under apps that are using it